Makefile

```diff
@@ -80,21 +81,21 @@ vet:
 fmt:
 	$(shell ./hack/update-gofmt.sh)
 
-container: $(HYPERFED_TARGET)-linux
-	cp -f $(HYPERFED_TARGET)-linux images/kubefed/hyperfed
+container: $(HYPERFED_TARGET)-linux-$(HOST_ARCH)
+	cp -f $(HYPERFED_TARGET)-linux-$(HOST_ARCH) images/kubefed/hyperfed
 	$(DOCKER) build images/kubefed -t $(IMAGE_NAME)
 	rm -f images/kubefed/hyperfed
 
 bindir:
 	mkdir -p $(BIN_DIR)
 
 COMMANDS := $(HYPERFED_TARGET) $(CONTROLLER_TARGET) $(KUBEFEDCTL_TARGET) $(WEBHOOK_TARGET)
-OSES := linux darwin
+OSES := linux-amd64 linux-arm64 linux-ppc64le linux-s390x darwin-amd64
```

Comment:
These aren't really just OSes anymore, they are OS + hardware architecture.

Reply:
It will make for a larger patch, but I can rename the variable if you prefer.

Reply:
Since you're modifying the variable's values already, I think it's okay to make the change as the variable is only used on line 101. Thanks!